nautobot/dcim/models/device_components.py

```diff
@@ -590,6 +612,7 @@ def to_csv(self):
             self.device.identifier if self.device else None,
             self.name,
             self.label,
+            self.parent_interface.name if self.parent_interface else None,
             self.lag.name if self.lag else None,
```

Comment:
Need to match `csv_headers` and `to_csv` fields...

```suggestion
            self.parent_interface.name if self.parent_interface else None,
            self.bridge.name if self.bridge else None,
            self.lag.name if self.lag else None,
```